tests/test_cases/test_array/test_array.py

```diff
@@ -25,30 +25,30 @@ def _check_type(tlog, hdl, expected):
 
 
 def _check_int(tlog, hdl, expected):
-    assert hdl.value == expected, "{!r}: Expected >{}< but got >{}<".format(
-        hdl, expected, hdl.value
-    )
+    assert (
+        hdl.value == expected
+    ), f"{hdl!r}: Expected >{expected}< but got >{hdl.value}<"
```

Comment:
We can bring this in, but honestly these should just be refactored to just the asserts.